bot/database.py

```diff
@@ -120,16 +120,19 @@ async def change_password(self, nickname: str, password: str, new_password: str,
     async def login(self, nickname: str, password: str, userid: str) -> bool:
         valid = await self.is_password_and_nickname_valid(nickname, password, userid)
         if valid:
-            self.logged_in_users[userid] = nickname
+            hashed_id = hashlib.sha256(bytes(userid, "utf-8"), usedforsecurity=True).hexdigest()
+            self.logged_in_users[hashed_id] = nickname
         return valid
     
     async def logout(self, userid: str) -> bool:
         if userid in self.logged_in_users:
-            self.logged_in_users.pop(userid)
+            hashed_id = hashlib.sha256(bytes(userid, "utf-8"), usedforsecurity=True).hexdigest()
+            self.logged_in_users.pop(hashed_id)
             return True
         return False
     
     async def get_nickname(self, userid: str) -> Optional[str]:
```

Comment:
Mas descriptive pa sana 'yong `get_nickname` but I can't think of an alternative for it.

Reply:
`get_logged_in_nickname`?

Reply:
wdyt HAHAHA

Reply:
masyadong verbose?

Reply:
tho changed all occurences lang naman ehh HAHAHAHAHAHA

Reply:
`get_nickname_from_sessions`?

Reply:
pede naman

Reply:
tho session singular dapat diba?